clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/TitanConfiguration.groovy

```diff
@@ -45,6 +46,12 @@ class TitanConfiguration {
   }
 
   @Bean
+  @DependsOn('netflixTitanCredentials')
+  TitanDeployHandler titanDeployHandler(TitanClientProvider titanClientProvider) {
```

Comment:
DependsOn can be brittle based on strings. If you put the bean type as a parameter input, it will ensure the other bean is created first, giving you a static way to declare the same dependency.